### ISSUE TITLE
Upgrade eth-trezor-keyring usages of @ethereumjs/tx and @ethereumjs/common

### ISSUE THREAD

**Issue by brad-decker** [MetaMask/metamask-extension#18028]

**Comment by danjm** [comment]
Currentlly, there are errors when building and running the extension with eth-trezor-keyring v1.0.0

These can be seen by running this branch: https://github.com/MetaMask/metamask-extension/pull/18062

These errors may have to be resolved to unblock this PR, or we may need to produce a new version of the trezor keyring that uses `trezor-connect` v8, while the deeper issues with v9 are resolved

**Comment by brad-decker** [comment]
@karliatto you are correct this is an issue from the etherumjs/tx library due to a dependency using wasm, we have solved this elsewhere. The only work remaining here is captured in #18030 (pulling the updated library into extension)
